src/test/ui/return/tail-expr-as-potential-return.rs

```diff
@@ -30,3 +29,19 @@ async fn bar(x: bool) -> Result<f64, i32> {
     }
     Ok(42.0)
 }
+
```

Comment:
Why the change to this test?

Reply:
i moved a normalize call so that the suggestion now works when theres a projection in the return type of an async fn. before this commit, it was missing. 

sorry, should've commented this drive-by.